kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardTransactionDetail.xml

```diff
@@ -106,6 +106,11 @@ http://rice.kuali.org/dd http://rice.kuali.org/dd/dd.xsd">
 </bean>
 
 <!-- Attribute Definitions -->
+    <bean id="ProcurementCardTransactionDetail-transactionOriginalCurrencyAmount" parent="ProcurementCardTransactionDetail-transactionOriginalCurrencyAmount-parentBean">
+        <property name="validationPattern">
+            <bean parent="FloatingPointValidationPattern" p:allowNegative="true" />
+        </property>
```

Comment:
transactionSalesTaxAmount is a required field, though the value may be 0.

When Enter Sales Tax Amount was implemented in UAF-2972, the settings to allow negative amounts are allowed by the base bean "GenericAttributes-genericAmount-parentBean" and the reference "GenericAmountValiationPattern"